clarity/src/vm/costs/mod.rs

```diff
@@ -1327,6 +1327,14 @@ impl ExecutionCost {
             read_length: first.read_length.max(second.read_length),
         }
     }
+
+    pub fn is_zero(&self) -> bool {
+        self.write_length == 0
+            && self.write_count == 0
+            && self.read_length == 0
+            && self.read_count == 0
+            && self.runtime == 0
```

Comment:
I would recommend doing:

```suggestion
        *self == Self::zero()
```

It's shorter, and this way if any fields are added to `ExecutionCost`, there's no chance of forgetting to update this function and it returning an incorrect value

Also declare the `zero()` function as `const`, which might help the compiler optimize this code better

Reply:
Good point. I guess a `ZERO` const would be even better than a `const` method, right?

Reply:
Refactored to use `ZERO` in 2e266a20622cb4722d0743be3cb180a8d5e2569e.

Reply:
I think it's better practice to use a constant, but this does make the diff bigger. Your call, either way is fine